index.html

```diff
@@ -16680,25 +16680,27 @@ <h3>Presentational Roles Conflict Resolution</h3>
             explicit role. If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined.
```

Comment:
Happy to move this to a separate issue to avoid bloating this PR, but does the following only apply to this second exception, or all of them?
> If the action of exposing the explicit role causes the accessibility tree to be malformed, the expected results are undefined.

Reply:
Pulled this out into a separate issue so this PR can proceed faster: #2638